src/include/OpenImageIO/errorhandler.h

```diff
@@ -83,16 +83,37 @@ class OIIO_API ErrorHandler {
     }
     virtual ~ErrorHandler() {}
 
+    /// Set desired verbosity level.
+    void verbosity(int v) { m_verbosity = v; }
+
+    /// Return the current verbosity level.
+    int verbosity() const { return m_verbosity; }
+
     /// The main (or "full detail") method -- takes a code (with high
     /// bits being an ErrCode) and writes the message, with a prefix
     /// indicating the error category (no prefix for "MESSAGE") and
     /// error string.
     virtual void operator()(int errcode, const std::string& msg);
 
-    /// Info message with printf-like formatted error message.
-    /// Will not print unless verbosity >= VERBOSE.
+    // Base cases -- take a single string
+    void info(const std::string& msg) { (*this)(EH_INFO, msg); }
+    void warning(const std::string& msg) { (*this)(EH_WARNING, msg); }
+    void error(const std::string& msg) { (*this)(EH_ERROR, msg); }
+    void severe(const std::string& msg) { (*this)(EH_SEVERE, msg); }
+    void message(const std::string& msg) { (*this)(EH_MESSAGE, msg); }
+#ifndef NDEBUG
+    void debug(const std::string& msg) { (*this)(EH_DEBUG, msg); }
+#else
+    void debug(const std::string&) {}
+#endif
+
+    //
+    // Formatted output with the same notation as Strutil::format.
+    /// Use with caution! Some day this will change to be fmt-like rather
+    /// than printf-like.
+    //
     template<typename... Args>
-    void info(string_view format, const Args&... args)
+    void info(const char* format, const Args&... args)
```

Comment:
Is there a reason not to use `Arg&&... args` and `std::forward` to get move semantics if appropriate?

It probably wouldn't matter in the vast majority of cases, but it might be a tiny bit more efficient if handling some temporary objects?

Reply:
I dunno. If that were so important, wouldn't fmt itself (and tinyformat before it) do its declarations like that?

Reply:
I do see what you're getting at here. Seems like the right place to use the "perfect forwarding" idiom. But then why is that not what's used in format and tinyformat themselves when they pass things around?

Reply:
I'll defer to the wisdom of those other projects. For `tinyformat` it might be due to its pre-C++11 compatibility, not sure about `fmt` ...

Reply:
I think it's also something that we can do later if we decide forwarding is better. It's just an inline template, changing to `&&` later won't break source or link compatibility. From the caller's point of view, the args are the same.

Reply:
I think I want to merge what I've got (which basically preserves the source call signatures we've had for a long time, but just adds new ones that can be counted on to stay explicitly printf-like), without having to do a lot more testing and soul-searching while I'm trying to get a release out. Then I will do more careful testing to see if there's advantage to switching to `&&`, which I think we can do any time without a second compatibility break.

Reply:
Sounds good.

Reply:
That makes perfect sense.